### PR TITLE
Release preparations

### PR DESCRIPTION
**Pull request by arogachev**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌

### CODE REVIEW

**Review by vjik — APPROVED** [review]

File: phpunit.xml.dist [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <phpunit
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
+    xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/10.5/phpunit.xsd"
```
Comment (roxblnfk) [inline phpunit.xml.dist:4]: it prefer to use the schema from the vendor dir

Comment (vjik) [reply phpunit.xml.dist:4]: Should we update it in package template also?

https://github.com/yiisoft/package-template/blob/master/phpunit.xml.dist

Comment (roxblnfk) [reply phpunit.xml.dist:4]: may be

Comment (samdark) [reply phpunit.xml.dist:4]: Yes. Definitely.

**Review by samdark — APPROVED** [review]

**Pull request merged**